### DISCUSSION TITLE
All set shortcut keys are invalid

### DISCUSSION THREAD

**Discussion by YFAnt** [wezterm/wezterm#4675] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20231203-124028-e3cd2e93

### Ask your question!

When I run the shortcut key, the expected effect does not appear, and I am always in the command line state without any response. Sometimes the computer also produces input error sound effects.

i use it in Apple M1  macos 13.4.1 (c) (22F770820d)

here is my config [config](https://github.com/YFAnt/dotfiles/tree/main/wezterm)

Can you help me figure out this problem? Thank you very much.

**Comment by bew** [comment]
Please run wezterm in a terminal with `debug_key_events = true`
https://wezfurlong.org/wezterm/config/lua/config/debug_key_events.html

And give us the logs of some keys that do not work as expected

**Reply by YFAnt** [reply]
I'm sorry for responding so late. In fact, it works, I made a mistake, Thank you very much for your help :)
